kythe/cxx/indexer/cxx/IndexerASTHooks.h

```diff
@@ -605,7 +605,7 @@ class IndexerASTVisitor : public clang::RecursiveASTVisitor<IndexerASTVisitor> {
   /// \brief Emit data for attributes attached to `Decl`, whose `NodeId`
   /// is `TargetNode`.
   void VisitAttributes(const clang::Decl* Decl,
-                       const GraphObserver::NodeId& TargetNode);
```

Comment:
IMO, `TargetNode` is the better spelling here (also what's mentioned in the preceding comment).